tests/integration/conftest.py

```diff
@@ -96,3 +98,24 @@ async def _get_unit_status_msg_list(application_name: str = "wordpress"):
 @pytest.fixture(scope="module", name="application_name")
 def fixture_application_name():
     return "wordpress"
+
+
+@pytest_asyncio.fixture(scope="function", name="default_admin_password")
+async def fixture_default_admin_password(ops_test: pytest_operator.plugin.OpsTest):
+    return_code, stdout, stderr = await ops_test.juju(
```

Comment:
There is a native way of doing this: https://pythonlibjuju.readthedocs.io/en/latest/api/juju.unit.html#juju.unit.Unit.run_action

The unit should be accessible under the `<wordpress application>.units[0]`

Reply:
Fixed, thanks!